tools.go

```diff
@@ -0,0 +1,3 @@
+package main
+
+import _ "github.com/benbjohnson/ego"
```

Comment:
I've seen this pattern as a way to track dependencies on command-line tools you need that do not show up in your source code